build.xml

```diff
@@ -1358,11 +1427,8 @@ ext = BeanUtils.cook(ext);
     <target name='server-war'>
         <ant dir='${server.dir}' target='war'/>
     </target>
-    <target name='taglib-jar'>
```

Comment:
It was part of cleanup work, we don't need to generate taglib-jar as part of webclient code
it is handled by ivy now